### PR TITLE
Document new VarCloner::setMinDepth function

### PR DESCRIPTION
**Pull request by james-johnston-thumbtack**
Documentation for new ``setMinDepth`` function in the VarDumper component's cloner.  (This function is pending code review / merge.)

### CODE REVIEW

**Comment by james-johnston-thumbtack** [comment]
The actual code has been merged, so this PR is also ready for review and subsequent merging now.

File: components/var_dumper/advanced.rst [hunk 1]
```diff
@@ -53,16 +53,23 @@ Before calling :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::cloneV
 you can configure these limits:
 
 :method:`Symfony\\Component\\VarDumper\\Cloner\\VarCloner::setMaxItems`
-    configures the maximum number of items that will be cloned
-    *past the first nesting level*. Items are counted using a breadth-first
+    Configures the maximum number of items that will be cloned
+    *past the minimum nesting depth*. Items are counted using a breadth-first
     algorithm so that lower level items have higher priority than deeply nested
-    items;
+    items. Specifying ``-1`` removes the limit.
```
Comment (HeahDude) [inline components/var_dumper/advanced.rst]: This paragraph changes should target 3.2 right?

Comment (james-johnston-thumbtack) [inline components/var_dumper/advanced.rst]: This paragraph still accurately describes behavior of older versions.  For versions < 3.4 we can assume the minimum nesting depth is hard-coded to 1; this is indicated by the next paragraph that says "The default value is 1, which is consistent with older Symfony versions."  The presence of `setMaxItems` is much older than v3.2 - it goes back to Symfony 2 (maybe back to the original introduction of VarCloner).

**Review by javiereguiluz — APPROVED** [review]

**Review by HeahDude — APPROVED** [review]

**Comment by javiereguiluz** [comment]
James, thanks for your contribution! I'm sorry it took us so long to merge it ... but it's finally merged!

**Comment by james-johnston-thumbtack** [comment]
Well, better late than never.  :)

**Pull request merged**